core/include/initcall.h

```diff
@@ -16,7 +16,7 @@ struct initcall {
 	int level;
 	const char *func_name;
 #endif
-};
+} __aligned(32);
```

Comment:
Please note that you are changing the arch independent file, we should not affect ARM side.